src/Tribe/Editor/Warnings.php

```diff
@@ -46,6 +47,40 @@ public function show_recurring_event_warning_message( $post_id ) {
 		$this->render_notice( $warning );
 	}
 
+	/**
+	 * Add Provider missing warning for tickets.
+	 *
+	 * @since TBD
+	 */
+	public function add_commerce_provider_warning() {
+		$available_modules = array_diff_key( \Tribe__Tickets__Tickets::modules(), [ 'Tribe__Tickets__RSVP' => true ] );
+
+		if ( count( $available_modules ) > 0 ) {
+			return;
+		}
+
+		$this->render_notice( $this->get_commerce_provider_missing_warning_message() );
+	}
+
+	/**
+	 * Get the Commerce Provider missing warning message.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The Commerce Provider missing message.
+	 */
+	public function get_commerce_provider_missing_warning_message() {
+		$kb_url      = 'http://m.tri.be/1ao5';
+
+		$link = sprintf(
+				'<a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s</a>',
+				esc_url( $kb_url ),
+				esc_html_x( '[Learn More]', 'Helper link in Ticket Editor', 'event-tickets' )
+		);
+
+		return sprintf( __( 'There is no payment gateway configured. To create tickets, you\'ll need to enable and configure an ecommerce solution. %1$s', 'event-tickets' ), $link );
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 330dfc66042566d0cb543ca1f83ec305</sub>